content/docs/user-guide/how-to/best-practices.md

```diff
@@ -0,0 +1,136 @@
+# Best Practices for DVC Projects
+
+Asking questions on data science collaboration to data scientists, engineers, or
+managers, we'll get a variety of answers. DVC provides a systematic approach
+towards managing and collaborating on data science projects. You can manage your
+projects with DVC more efficiently using the practices listed here:
+
+- Source code and data versioning
+
+  You can use DVC to avoid discrepancies between
```

Comment:
These bullets should probably be H2 headers instead, so each practice get's a right-hand nav entry. The doc is too long for it to be a single bullet list anyway.